src/components/organize/tasks/forms/typeConfigFields/VisitLinkFields.tsx

```diff
@@ -0,0 +1,19 @@
+import { TextField } from 'mui-rff';
+import { useIntl } from 'react-intl';
+
+import { VISIT_LINK_FIELDS } from '../constants';
+
+const VisitLinkFields = (): JSX.Element => {
+    const intl = useIntl();
+    return (
+        <TextField
```

Comment:
Maybe we could have some sort of URL validation on this? Lets file a separate low-priority issue for that.

Reply:
Filed as #345